src/components/ModalInput/ModalInput.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Text, Button, Input, InputProps, Flex, Link } from '@pancakeswap-libs/uikit'
+import useI18n from '../../hooks/useI18n'
+
+interface ModalInputProps {
+  max: string
+  symbol: string
+  onSelectMax?: () => void
+  onChange: (e: React.FormEvent<HTMLInputElement>) => void
+  placeholder?: string
+  value: string
+  addLiquidityUrl?: string
+  withdrawal?: boolean
+}
+
+const getBoxShadow = ({ isWarning = false, theme }) => {
```

Comment:
A copy & paste job from the uikit input - it could maybe be exported and used here instead.